### PR TITLE
no-ticket refactoring to use logrus hooks

### PR DESCRIPTION
**Pull request by fabioesposito**

### CODE REVIEW

File: ranger_logger/logrus_hooks.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	"fmt"
+	"net"
+
+	"github.com/bshuster-repo/logrus-logstash-hook"
+	"github.com/johntdyer/slackrus"
+	"github.com/sirupsen/logrus"
+)
+
+// NewLogstashHook ...
+func NewLogstashHook(protocol string, addr string, f Formatter) Hook {
```
Comment (bubleg) [inline ranger_logger/logrus_hooks.go]: I love how with every next argument the variable names are getting shorter :D

File: ranger_logger/logrus_hooks.go [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	conn, err := net.Dial(protocol, addr)
+	if err != nil {
+		fmt.Printf("Unable to connect to logstash: %s \n", err.Error())
+		return nil
+	}
+	return logrustash.New(conn, f)
+}
+
+// NewSlackHook constructor
+func NewSlackHook(channel string, webhook string, logNotificationLevel string) Hook {
```
Comment (bubleg) [inline ranger_logger/logrus_hooks.go]: `channel, webhook, logNotificationLevel string`?

File: ranger_logger/logrus_hooks.go (hunk 1 above)
Comment (fabioesposito) [inline ranger_logger/logrus_hooks.go]: :D fixed

File: ranger_logger/logrus_hooks.go (hunk 2 above)
Comment (fabioesposito) [inline ranger_logger/logrus_hooks.go]: cool didnt know that

**Pull request merged**